nixos/modules/security/pam.nix

```diff
@@ -46,6 +46,16 @@ let
         '';
       };
 
+      google2FA = mkOption {
```

Comment:
Looks good to me. Would be `googleAuthenticator` be more self-explanatory then `google2FA`?

Reply:
Considering that there are [other config settings](https://github.com/google/google-authenticator-libpam/blob/master/README.md#module-options) that we might want to support later, it might make sense to "namespace" the config option like so:

```
google2FA = {
  enable = mkOption {
    default = false;
    type = types.bool;
    description = ''
      If set, users with enabled Google Authenticator (created
      <filename>~/.google_authenticator</filename>) will be required
      to provide Google Authenticator token to log in.
    '';
  };
};
```

Reply:
that makes sense.

Reply:
The additional namespace is a very good idea.

`authenticator` seemed like a longish word. I have very little idea about conventions here. I am happy to change it.

Reply:
For me it was more about being consistent with upstream.